packages/telemetry/README.md

```diff
@@ -6,6 +6,8 @@ This package is a convenience wrapper built around [open telemetry](https://open
 
 Telemetry in Cypress is disabled by default. To enable telemetry in Cypress set `CYPRESS_INTERNAL_ENABLE_TELEMETRY="true"`.
 
+Verbose telemetry in Cypress is disabled by default when telemetry is enabled. To enable verbose telemetry in Cypress set `CYPRESS_INTERNAL_ENABLE_TELEMETRY_VERBOSE="true"`. This will enable telemetry for areas of the code that report a lot of events, such as the `@packages/proxy` code.
```

Comment:
if `CYPRESS_INTERNAL_ENABLE_TELEMETRY_VERBOSE` is set to `"true"`, should we also imply that `CYPRESS_INTERNAL_ENABLE_TELEMETRY` is true so we don't have to pass in both flags?

Reply:
Seems reasonable

Reply:
addressed